src/index.html

```diff
@@ -1,17 +1,245 @@
+              autocomplete="off"
+              id="name"
+              name="name"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+
+            <input
+              type="number"
+              id="age"
+              min="1"
+              max="100"
+              value="12"
+              name="age"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="date">
+            Full date of birth:
+
+            <input
+              type="date"
+              id="date"
+              name="date"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="agreement">
+            I accept the term of the agreement
+
+            <input
+              type="checkbox"
+              name="agreement"
+              id="agreement"
+              required
+            >
+
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="password">
+            Password:
+
+            <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="6"
+              maxlength="15"
+            >
+
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label for="cats">
+            Do you love cats?
+
+            <input
+              type="radio"
+              name="cats"
+              id="catsYes"
+            >
+            Yes
+
+            <input
+              type="radio"
+              name="cats"
+              id="catsNo"
+            >
+            No
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+
+            <input
+              type="color"
+              name="color"
+              id="color"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="sleep">
+            What time do you go to bed?
+
+            <input
+              type="time"
+              name="sleep"
+              id="sleep"
+            >
+
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="cars">
+            What are your favorite brand of cars?
+
+            <select
+              name="cars"
+              id="cars"
+              multiple>
```

Comment:
Fix formating for this select tag, add indentations like:
```suggestion
            <select
              name="cars"
              id="cars"
              multiple
            >
```